packages/grafana-ui/src/components/Forms/Form.mdx

```diff
@@ -182,6 +182,8 @@ const defaultValues: FormDto {
 </Forms.Form>
 ```
 
+### Validate
+
```

Comment:
Let's add this docs in a separate PR not to block this one

Reply:
Ops, too late :/

Reply:
no worries 🗡